.travis.yml

```diff
@@ -3,10 +3,19 @@ language: python
 python:
   - 2.6
   - 2.7
+  - 3.4
 
 env:
   - DJANGO_VERSION=1.5
   - DJANGO_VERSION=1.6
+  - DJANGO_VERSION=1.7
+  - DJANGO_VERSION=1.8
+matrix:
+  exclude:
+    - python: 2.6
+      env: DJANGO_VERSION=1.7
+    - python: 2.6
+      env: DJANGO_VERSION=1.8
```

Comment:
I think you should exclude old versions of django with python 3 (see django-geojson repo)